litmus-portal/frontend/src/components/Button/BackButton/index.tsx

```diff
@@ -1,23 +1,18 @@
-import { Button, Typography } from '@material-ui/core';
+import { IconButton, IconButtonProps, Typography } from '@material-ui/core';
 import React from 'react';
 import useStyles from './styles';
 
-interface BackButtonProps {
-  isDisabled: boolean;
-}
-
-const BackButton: React.FC<BackButtonProps> = ({ isDisabled }) => {
+const BackButton: React.FC<IconButtonProps> = ({ onClick }) => {
   const classes = useStyles();
   return (
-    <Button
+    <IconButton
       size="medium"
       className={classes.btn}
-      disabled={isDisabled}
-      onClick={() => window.history.back()}
+      onClick={onClick ?? (() => window.history.back())}
```

Comment:
```suggestion
      onClick={onClick ?? (() => history.goBack())}
```
While the window object is definitely more performant it being browser native, shouldn't we try to stick with react-router where we can?